### PR TITLE
Create new_bug.md

### PR DESCRIPTION
**Pull request by abbylindsay**

### CODE REVIEW

**Review by marcderhammer — APPROVED** [review]

**Review by andrewalc — APPROVED** [review]

**Review by j93lim — APPROVED** [review]

**Review by rcole1025 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/new_bug.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+name: New Bug
+about: Identification of a new issue
+title: 'New Bug'
+labels:
```
Comment (barchard) [inline .github/ISSUE_TEMPLATE/new_bug.md:5]: We should add use a standard label.

File: .github/ISSUE_TEMPLATE/new_bug.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+name: New Bug
```
Comment (barchard) [inline .github/ISSUE_TEMPLATE/new_bug.md:2]: How does this differ than the built-in bug report template?

File: .github/ISSUE_TEMPLATE/new_bug.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+---
+
+**Describe the bug- why is it wrong?:**
+<!-- A clear and concise description of what you/the participant is seeing and why it is a problem. -->
+
+**How was this bug identified:**
+<!-- Who saw it? What was the context? -->
+
+**How to reproduce?:**
```
Comment (barchard) [inline .github/ISSUE_TEMPLATE/new_bug.md:16]: We should make fields required as appropriate. https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

File: .github/ISSUE_TEMPLATE/new_bug.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+**Troubleshooting completed so far:**
+<!-- What has been done thus far to resolve/investigate the bug? -->
+
+**Expected behavior?:**
+<!-- Be descriptive: what should be happening? Provide examples if possible. -->
+
+**Screenshot(s):**
+<!-- Show the bug. -->
+
+**Priority level?:**
```
Comment (barchard) [inline .github/ISSUE_TEMPLATE/new_bug.md:28]: This should be a drop down of predefined items.

```
- type: dropdown
    id: version
    attributes:
      label: Version
      description: What version of our software are you running?
      options:
        - 1.0.2 (Default)
        - 1.0.3 (Edge)
    validations:
      required: true
```

File: .github/ISSUE_TEMPLATE/new_bug.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+**How was this bug identified:**
+<!-- Who saw it? What was the context? -->
+
+**How to reproduce?:**
+<!-- If no instructions, you MUST provide an explaination as to why not. -->
+
+**Troubleshooting completed so far:**
+<!-- What has been done thus far to resolve/investigate the bug? -->
+
+**Expected behavior?:**
```
Comment (barchard) [inline .github/ISSUE_TEMPLATE/new_bug.md:22]: Usually this is expected behavior and actual behavior. Maybe should match the columns and terms used in the V&V

File: .github/ISSUE_TEMPLATE/new_bug.md (hunk 4 above)
Comment (barchard) [reply .github/ISSUE_TEMPLATE/new_bug.md:28]: [Reference](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms)

File: .github/ISSUE_TEMPLATE/new_bug.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+<!-- Be descriptive: what should be happening? Provide examples if possible. -->
+
+**Screenshot(s):**
+<!-- Show the bug. -->
+
+**Priority level?:**
+<!-- 1- a V&V will not be run for this issue alone. 2- Want to have 3- Causing minor impact to study/participant 4- Causing big impact to study/participant 5- BAD BAD BAD -->
+
+**Additional Context:**
```
Comment (barchard) [inline .github/ISSUE_TEMPLATE/new_bug.md:31]: Capture the Tablet IDs? Parse Account(s)? Sensor serial number(s)?

File: .github/ISSUE_TEMPLATE/new_bug.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+<!-- The more info the better -->
```
Comment (barchard) [inline .github/ISSUE_TEMPLATE/new_bug.md:32]: Do you want to capture a "Component" dropdown here? E.g., "Sensor", "Reports", "Portal", "Tablet"?

File: .github/ISSUE_TEMPLATE/new_bug.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
+---
+name: New Bug
+about: Identification of a new issue
+title: 'New Bug'
+labels: 
+assignees: '@abbylindsay'
+
+---
```
Comment (barchard) [inline .github/ISSUE_TEMPLATE/new_bug.md:8]: We usually want to track the version information of the bug was identified in. For example, app version 1.2 if know. Or firmware 4.0.1.

**Review by barchard — CHANGES_REQUESTED** [review]
We should also capture severity of bug as a drop down. E.g., Blocker. This is good:

https://www.browserstack.com/guide/bug-severity-vs-priority

File: .github/ISSUE_TEMPLATE/new_bug.md (hunk 2 above)
Comment (abbylindsay) [reply .github/ISSUE_TEMPLATE/new_bug.md:2]: i has some questions based on built in bug template but has added questions based on what our internal needs are

File: .github/ISSUE_TEMPLATE/new_bug.md (hunk 3 above)
Comment (abbylindsay) [reply .github/ISSUE_TEMPLATE/new_bug.md:16]: at first glance it looks like the issue templates would have to be added to each repo to do this... is that worth it? ill look at it more

File: .github/ISSUE_TEMPLATE/new_bug.md (hunk 4 above)
Comment (abbylindsay) [reply .github/ISSUE_TEMPLATE/new_bug.md:28]: we can have a drop down. rylee, marc, james and I had a thorough convo defining what type of issue is what level (1-5) that would be callibrated with the larger group

File: .github/ISSUE_TEMPLATE/new_bug.md (hunk 7 above)
Comment (abbylindsay) [reply .github/ISSUE_TEMPLATE/new_bug.md:32]: do you think we need that?  i think that would be clear by the issue. i dont want ops to have an exhaustive report each time they fill out an issue request (yes a drop down is easy but i think its the principle here of deciding what is actually a value-add question)

File: .github/ISSUE_TEMPLATE/new_bug.md (hunk 4 above)
Comment (barchard) [reply .github/ISSUE_TEMPLATE/new_bug.md:28]: yes, let's remove any ambiguity here and make this error proof and standardized.

Comment (barchard) [reply .github/ISSUE_TEMPLATE/new_bug.md:28]: We should also define what each of the levels, etc. *mean* so it's clear.

**Comment by abbylindsay** [comment]
> We should also capture severity of bug as a drop down. E.g., Blocker. This is good:
> 
> https://www.browserstack.com/guide/bug-severity-vs-priority

we can change the title to severity level instead of priority level - thats the information being captured there. priority was just the languaged used before.

**Comment by barchard** [comment]
Severity and priority are 2 different items. I think we may want to ask both. It may be a high priority item (e.g., deadline approaching) but low severity (e.g., wording change or typo)

File: .github/ISSUE_TEMPLATE/new_bug.md (hunk 4 above)
Comment (abbylindsay) [reply .github/ISSUE_TEMPLATE/new_bug.md:28]: this is listed under priority level to define. as i said, rylee, james, marc and I spent a lot of time thinking of a scale that we were going to share with the group so everyone is callibrated. the short version is listed here as a reminder.

**Review by rcole1025 — APPROVED** [review]

**Review by barchard — APPROVED** [review]

**Pull request merged**